extensions/binding/org.eclipse.smarthome.binding.tradfri/README.md

```diff
@@ -4,23 +4,35 @@ This binding integrates the IKEA TRÅDFRI gateway and devices connected to it (s
 
 ## Supported Things
 
-Besides the gateway (thing type "gateway"), the binding currently supports dimmable warm white bulbs as well as white spectrum bulbs.
-
-The thing type ids are defined according to the lighting devices defined for ZigBee LightLink ([see page 24, table 2](https://www.nxp.com/documents/user_manual/JN-UG-3091.pdf). These are:
-
-| Device type              | ZigBee Device ID | Thing type |
-|--------------------------|------------------|------------|
-| Dimmable Light           | 0x0100           | 0100       |
-| Colour Temperature Light | 0x0220           | 0220       |
-| Extended Colour Light    | 0x0210           | 0210       |
+Beside the gateway (thing type "gateway"), the binding currently supports colored bulbs, dimmable warm white bulbs as well as white spectrum bulbs.
+The binding also supports read-only data from remote controls and motion sensors (e.g. the battery status).
+The TRÅDFRI controller and sensor devices currently cannot be observed right away.
+We assume that they are communicating directly with the bulbs or lamps without routing their commands through the gateway.
+This makes it nearly impossible to trigger events for pressed buttons.
+We only can access some static data like the present status or battery level. 
+
+The thing type ids are defined according to the lighting devices defined for ZigBee LightLink ([see page 24, table 2](https://www.nxp.com/documents/user_manual/JN-UG-3091.pdf).
+These are:
+
+| Device type                     | ZigBee Device ID | Thing type |
+|---------------------------------|------------------|------------|
+| Dimmable Light                  | 0x0100           | 0100       |
+| Colour Temperature Light        | 0x0220           | 0220       |
+| Extended Colour Light           | 0x0210           | 0210       |
+| Occupancy Sensor Unit           | 0x0107           | 0107       |
```

Comment:
Where did you get the 0x0107 from? I cannot find it in the spec referenced above.

Reply:
see https://www.nxp.com/docs/en/user-guide/JN-UG-3114.pdf (page 71ff)

Reply:
Might be worth to update the link above then as well!